client/post-editor/editor-deprecation-dialog/index.jsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import isEditorDeprecationDialogShowing from 'state/selectors/is-editor-deprecation-dialog-showing';
+import { hideEditorDeprecationDialog } from 'state/ui/editor-deprecation-dialog/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import { localize } from 'i18n-calypso';
+import { composeAnalytics, recordTracksEvent, withAnalytics } from 'state/analytics/actions';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import InlineSupportLink from 'components/inline-support-link';
+import { localizeUrl } from 'lib/i18n-utils';
+import FormattedDate from 'components/formatted-date';
+
+/**
+ * Style dependencies
+ */
+import '@wordpress/components/build-style/style.css';
+import './style.scss';
+
+class EditorDeprecationDialog extends Component {
+	static propTypes = {
+		// connected properties
+		translate: PropTypes.func,
+		gutenbergUrl: PropTypes.string,
+		hideDialog: PropTypes.func,
+		optIn: PropTypes.func,
+		logNotNow: PropTypes.func,
+		siteId: PropTypes.number,
+		isDialogShowing: PropTypes.bool,
+	};
+
+	notNow = () => {
+		const { logNotNow, hideDialog } = this.props;
+		logNotNow();
+		hideDialog();
+	};
+
+	optInToGutenberg = () => {
+		const { gutenbergUrl, optIn, siteId, hideDialog } = this.props;
+		hideDialog();
+		optIn( siteId, gutenbergUrl );
+	};
+
+	render() {
+		const { isDialogShowing, translate } = this.props;
+
+		if ( ! isDialogShowing ) {
+			return null;
+		}
+
+		return (
+			<Modal
+				title={ translate( 'The Block Editor is coming.' ) }
+				className="editor-deprecation-dialog"
+				onRequestClose={ this.notNow }
+				isDismissible={ false }
+				shouldCloseOnClickOutside={ false }
+			>
+				<div className="editor-deprecation-dialog__illustration" />
+
+				<p className="editor-deprecation-dialog__subhead">
+					{ translate(
+						'Get a head start before we enable it for everyone on {{date/}}. {{support}}Read more{{/support}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **3** times:
`translate( 'Get a head start before we activate it for everyone on {{date/}}. {{support}}Read more{{/support}}.' )` *ES Score: 8*